src/pages/aurora-site/social-media-course.jsx

```
@@ -0,0 +1,263 @@
+                    (objective, index) => (
+                      <li key={index} className="flex items-start">
+                        <span className="text-light-blue-1 mr-2">•</span>{" "}
+                        {objective}
+                      </li>
+                    )
+                  )}
+                </ul>
+              </div>
+              <div>
+                <h3 className="font-medium text-neutral-1 mb-2">
+                  Skills You'll Gain
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)